panel/src/components/Forms/Input/SlugInput.vue

```diff
@@ -53,7 +53,7 @@ export default {
   data() {
     return {
       slug: this.sluggify(this.value),
-      slugs: this.$languages.current ? this.$languages.current.rules : this.$system.slugs,
+      slugs: window.panel.$language ? window.panel.$language.rules : this.$system.slugs,
```

Comment:
`this.$language` and `this.$language.rules` don't work here?

Reply:
`this.$language` works. When should we choose `this.$..` over `window.panel`? 🙂